Lib/test/test_email/test_parser.py

```diff
@@ -30,6 +30,23 @@ def test_custom_message_gets_policy_if_possible_from_file(self):
         self.assertIsInstance(msg, self.MyMessage)
         self.assertIs(msg.check_policy, self.MyPolicy)
 
+    def test_group_missing_semicolon_from_file(self):
+        email_message = ('From: Pete <pete@foo.bar>\r\n'
+                         'To: A Group:Chris Jones <c@a.test>,'
+                         'joe@where.test,John <jdoe@one.test>\r\n'
+                         'Cc: Undisclosed recipients:;\r\n'
+                         'Date: Thu, 13 Feb 1969 23:32:54 -0330\r\n'
+                         'Message-ID: <testabcd.1234@foo.bar>\r\n\r\n'
+                         'Testing.')
+        source_file = io.StringIO(email_message)
+        msg = email.message_from_file(source_file, policy=default)
+        to_header = msg['To']
+        addresses = to_header.addresses
+        self.assertEqual(len(addresses), 3)
+        self.assertEqual(addresses[0].display_name, 'Chris Jones')
+        self.assertEqual(addresses[1].username, 'joe')
+        self.assertEqual(addresses[2].display_name, 'John')
```

Comment:
I'd prefer a test in test__header_value_parser, since that's the module that's being changed.  Specifically, if you search the file for "# get_group" you'll see the set of tests for the function this PR modifies.  Put the test in there.